extensions-core/kinesis-indexing-service/src/main/java/org/apache/druid/indexing/kinesis/KinesisRecordSupplier.java

```diff
@@ -366,8 +390,8 @@ private Runnable fetchRecords()
           log.error(e, "encounted AWS error while attempting to fetch records, will not retry");
           throw e;
         }
-        catch (AmazonServiceException e) {
```

Comment:
Are there no tests that cover this path? I'm surprised that we didn't need to make an associated test change somewhere because of this and the change on line 815

Reply:
AmazonServiceException is a sub-class of AmazonClientException so previous tests would work as it is.